router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
I think we should split off the ports, so that we can keep the pair of 80/443 linked together, allow 80 to be disabled, and have a separate set of additional HTTP/HTTPS ports like so:

```text
DEFAULT_HTTP_PORT=80 # can also be 0 for disabled
DEFAULT_HTTPS_PORT=443
ADDITIONAL_HTTP_PORTS=123,456
ADDITIONAL_HTTPS_PORTS=678,901
```